### DISCUSSION TITLE
Integrate deep learning based feature extraction and maching methods

### DISCUSSION THREAD

**Discussion by zhouyu1214** [OpenStitching/stitching#109] [Ideas]
Hi,
Thank you for your excellent package, your package really helped a lot and much more faster than other methods I tried.

I'm trying to use some deep learning based feature extraction and maching methods，like:
super point: https://github.com/magicleap/SuperPointPretrainedNetwork
super glue:https://github.com/magicleap/SuperGluePretrainedNetwork

but having trouble integration these model with the package, maybe because the features and matches  I got from the deep learning methods have different data structure.

Do you have any suggestion？

Thank you for your time.

**Comment by lukasalexanderweber** [comment]
Thanks for reaching out, glad it helped! The repos look nice, and yeah I think you would need to find a way to convert features and matches into the format used by opencv stitching module.

**Reply by neriniccolodei** [reply]
@zhouyu1214 hi, have you investigated this problem further and may share some progress? I am on this as well. Kind regards
